lib/core/viewmodels/more_viewmodel.dart

```diff
@@ -66,11 +71,15 @@ class MoreViewModel extends FutureViewModel {
     } on Exception catch (e) {
       onError(e);
     }
-    UserRepository().logOut();
+    await PreferencesService().clear();
+    await UserRepository().logOut();
+
```

Comment:
It also necessary to invalidate the contents of CourseRepository.coursesActivity and sessions. A simple clear should be enough :sweat_smile:

Reply:
done